api/package-lock.json

```diff
@@ -0,0 +1,96 @@
+{
```

Comment:
this file should not be in the `api/` folder. Actually, maybe it should not be commited at all, I think you have to decide between NPM or Yarn. Yarn uses `yarn.lock`, NPM uses `package-lock.json`. There should be only one of the two.

Reply:
That's weird, I created the project with npm only.